packages/elements/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@clerk/elements",
+  "version": "0.0.1",
+  "description": "Clerk Elements ",
+  "keywords": [
+    "clerk",
+    "typescript",
+    "nextjs",
+    "auth",
+    "authentication",
+    "passwordless",
+    "session",
+    "jwt"
+  ],
+  "homepage": "https://clerk.com/",
+  "bugs": {
+    "url": "https://github.com/clerk/javascript/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/clerk/javascript.git",
+    "directory": "packages/elements"
+  },
+  "license": "MIT",
+  "author": "Clerk",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "npm run clean && tsup",
+    "clean": "rimraf ./dist",
+    "dev": "tsup --watch",
+    "dev:publish": "npm run dev -- --env.publish",
+    "lint": "eslint src/",
+    "lint:attw": "attw --pack .",
+    "lint:publint": "publint",
+    "test": "jest --passWithNoTests",
+    "test:cache:clear": "jest --clearCache --useStderr"
+  },
+  "dependencies": {
+    "@clerk/clerk-react": "*"
+  },
+  "devDependencies": {
+    "@types/node": "^18.17.0",
+    "@types/react": "*",
+    "@types/react-dom": "*",
+    "eslint-config-custom": "*",
+    "next": "^13.4.19",
+    "tslib": "2.4.1",
+    "typescript": "*"
+  },
+  "peerDependencies": {
+    "next": ">=10",
```

Comment:
I don't think Next is required for Elements?

So we should make this optional:

```json
"peerDependenciesMeta": {
  "next": {
    "optional": true
  }
},
```